src/Tailwind/tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
is this different with tailwind.config.js in root directory?

Reply:
please check is there any duplicate problem with this and the config in the root folder